manager/manager.go

```diff
@@ -59,11 +66,21 @@ type syncData struct {
 	mutex sync.Mutex
 }
 
-func NewManager(sources []source_api.Source, sinkManager sinks.ExternalSinkManager, res, bufferDuration time.Duration, align bool) (Manager, error) {
+func NewManager(sources []source_api.Source, sinkManager sinks.ExternalSinkManager, res, bufferDuration time.Duration, useModel bool, modelRes time.Duration, align bool) (Manager, error) {
+	// TimeStore constructor passed to the cluster implementation.
+	// TODO(alex): determine default analogy of cache duration to Timestore durations.
```

Comment:
default analogy means the ratio of model_duration to cache_duration (passed as bufferDuration to NewManager). As of now, model_duration = 5 * cache_duration for no specific reason. However, other options include using a flag to determine the exact duration, while defaulting to a multiple of cache_duration.